lectures/opt_tax_recur.md

```diff
@@ -31,7 +31,6 @@ In addition to what's in Anaconda, this lecture will need the following librarie
 tags: [hide-output]
 ---
 !pip install --upgrade quantecon
-!pip install interpolation
```

Comment:
This is need because of lecture specific static content that we have: https://github.com/QuantEcon/lecture-dps/blob/fa2e6932ac9b3222d05595eaf7def9de5961f424/lectures/_static/lecture_specific/opt_tax_recur/recursive_allocation.py.

Reply:
@mmcky I don't have push access on this repo. But this should be the fix.

Reply:
thanks @kp992 migrated that file from python-advanced